### ISSUE TITLE
User Story: Displaying recent batches table

### ISSUE THREAD

**Issue by jonathanbobel** [GSA/notifications-admin#1097]
### As a [type of user],

System Admin

### I want [an action or feature],

To review what types of messages different people on my team are sending

### So that [benefit or value].

They can get a snapshot of what's going on within a service

### Detailed condition or criteria that must be met for the user story to be considered complete.

- The system admin should be able to view, in table format, recent and historical batches to gain insights 
- The system admin should be able to access a **reports** area and be able to download reports based on batched jobs

### List of specific tasks or sub-tasks that need to be done to implement the user story.

- [x] #1114
This table is on the dashboard landing page, but without search and truncated (maybe 5-10 rows with a link to the full page
&nbsp;&nbsp;- Uploaded file name
&nbsp;&nbsp;- Message template
&nbsp;&nbsp;- Sender
&nbsp;&nbsp;- Time sent
&nbsp;&nbsp;- Number of recipients 

- [x] #1124

- [ ] Create a separate page that holds a searchable, paginated table (future feature)

### List any dependencies that need to be resolved before starting or completing this user story.

- Creating a batch table with the necessary columns (maybe we have this information already?)

### Any additional information or context that might be useful for the team.

Connected to EPIC: #1108

**Comment by em-herrick** [comment]
`The system admin should be able to access a reports area and be able to download reports based on batched jobs`

--> i like this, but i don't think it will be possible with our 7 day data retention policy, we might need to remove

**Comment by jonathanbobel** [comment]
Refer to the _recent campaign_ area as a sample for the batch table on the dashboard landing page

![Image](https://github.com/GSA/notifications-admin/assets/1233986/d45b6205-99b5-48db-8be5-4609f4c5ff7f)

**Comment by jonathanbobel** [comment]
![Image](https://github.com/GSA/notifications-admin/assets/1233986/26a913d2-1291-40c8-acc9-eb3c61b5520b)

**Comment by jonathanbobel** [comment]
![Image](https://github.com/GSA/notifications-admin/assets/1233986/d313317e-f851-4a8b-8ee8-e8b7e088a770)

**Comment by em-herrick** [comment]
@maomeara63 Would love your brain on the table headers before @heyitsmebev picks up this again to add additional columns to her work already completed. Feel free to edit the task section above directly if you have suggested edits.

**Comment by tdlowden** [comment]
@jonathanbobel @heyitsmebev would we consider this completed?

**Comment by heyitsmebev** [comment]
@tdlowden Yes, I would consider completed. We could created another ticket in the future if decide to build out additional features such as search...etc
